src/components/02-organisms/RajoitteetList/index.js

```diff
@@ -5,15 +5,20 @@ import SimpleButton from "components/00-atoms/SimpleButton";
 import { Typography } from "@material-ui/core";
 import { getRajoiteListamuodossa } from "utils/rajoitteetUtils";
 import HtmlContent from "components/01-molecules/HtmlContent";
+import { useIntl } from "react-intl";
+import rajoitteetMessages from "i18n/definitions/rajoitteet";
 
 const RajoitteetList = ({
   locale,
   onModifyRestriction,
   onRemoveRestriction,
   rajoitteet
 }) => {
+  const { formatMessage } = useIntl();
   if (isEmpty(rajoitteet)) {
-    return <p>Ei rajoitteita.</p>;
+    return (
+      <p className="mt-6">{formatMessage(rajoitteetMessages.eiRajoitteita)} </p>
```

Comment:
Tiketissä oli mainittu että napin ja ohjetekstin väli on liiansuuri eli siinä on 1.5em liikaa tyhjää. Sitä ei ole nyt korjattu tässä vaan tämä siirtää napin alapuolella olevaa "Ei rajoitteita" tekstiä 1.5em alaspäin (Varmaan tämäkin on hyvä olla mukana korjauksessa, vaikka ei mainittu tiketillä).